doc/ale-supported-languages-and-tools.txt

```diff
@@ -183,6 +183,7 @@ Notes:
   * `goimports`
   * `golangci-lint`!!
   * `golangserver`
+	* golines
```

Comment:
This one took me a while to figure out. The indentation should be using spaces instead of tabs and is missing the backticks **\`golines\`**

Reply:
My bad, for some reason vim was hiding the backticks around the other tool names and I didn't know they were required.